.github/workflows/php.yml

```diff
@@ -28,4 +31,15 @@ jobs:
       run: composer lint:check
       
     - name: PHPUnit Tests
-      run: composer phpunit
+      run: vendor/bin/phpunit --coverage-clover coverage.xml
+
+    - name: Generate code coverage badge
+      run: vendor/bin/php-coverage-badger coverage.xml .github/badges/coverage.svg
+
+    - name: Commit code coverage badge
+      uses: EndBug/add-and-commit@v5
+      with:
+        message: "Commit backend coverage badge"
```

Comment:
You should use `[ci skip] feat: Commit backend coverage badge`
To not run the pipeline again,
Reference: 
- GitHub Skip workflow https://docs.github.com/pt/actions/managing-workflow-runs/skipping-workflow-runs
- LinioIT commit messages: https://it-wiki.linio.com/standards/commit/

Reply:
That's why we wanted the review from someone from Infra, thanks @felipegirotti

Reply:
That's why we wanted the review from someone from Infra, thanks Felipe.

Reply:
thanks, I will fix it!